model/metrics/hardware/common.yaml

```diff
@@ -0,0 +1,59 @@
+groups:
+  # COMMON METRICS
+  - id: metric.hardware.attributes
+    type: attribute_group
+    brief: "Attributes for hardware metrics"
+    extends: hardware.attributes.common
+    attributes:
+      - ref: hw.type
+        requirement_level: required
+
+  - id: metric.hardware.energy
+    type: metric
+    metric_name: hw.energy
+    stability: experimental
+    brief: "Energy consumed by the component, in joules"
+    instrument: counter
+    unit: "J"
+    extends: metric.hardware.attributes
+
+  - id: metric.hardware.errors
+    type: metric
+    metric_name: hw.errors
+    stability: experimental
+    brief: "Number of errors encountered by the component"
+    instrument: counter
+    unit: "{errors}"
+    extends: metric.hardware.attributes
+    attributes:
+      - ref: hw.error.type
```

Comment:
```suggestion
      - ref: error.type
```

Reply:
I have answered in another similar comment

Reply:
Since there was no defined attribute for `hw.error.type`, my strong preference would be not to introduce one and instead use generic `error.type`.

Otherwise, we'd end up with another attribute the should be immediately deprecated.